test/notification.model.test.js

```diff
@@ -30,13 +32,14 @@ var Notification = models.Notification;
 /**
  * Tests.
  */
-describe("notification.model.test.js", () => {
+describe.only("notification.model.test.js", () => {
```

Comment:
@asood123 I have noticed multiple things in that code:
- The email should be sent with `sendMessageFromActivity` but this function does not exist
- The notifications lib expects `notifConfigs` to be able to send the email. In the tests, this array is empy, I think that's why it doesn't get called

Once those two things are fixed, I think that we will be able to test with a stub.